src/Keboola/S3Extractor/S3ExceptionConverter.php

```diff
@@ -70,4 +74,18 @@ private static function handleBaseUserErrors(S3Exception $e): void
         }
         throw new Exception($e->getMessage());
     }
+
+    /**
+     * @param S3Exception $e
+     * @param string $searchKey
+     * @throws Exception
+     */
+    private static function handleNotFound(S3Exception $e, string $searchKey): void
+    {
+        if ($e->getAwsErrorCode() === self::ERROR_CODE_NOT_FOUND_KEY) {
+            throw new Exception(sprintf('Error 404: Key `%s` not found.', $searchKey), $e->getCode(), $e);
```

Comment:
```suggestion
            throw new Exception(sprintf('Error 404: Key "%s" not found.', $searchKey), $e->getCode(), $e);
```

myslim, ze jsme tak nejak dospeli k tomu, ze uz davame vsude hodnoty do dvojitejch uvozovek :)

Reply:
Ok, změněno, budu si pro příště pamatovat :)